app/src/main/res/layout/migration_list_item.xml

```diff
@@ -27,7 +27,7 @@
             android:layout_marginStart="24dp"
             android:layout_marginEnd="32dp"
             android:textColor="@color/dark_grey_90"
-            android:fontFamily="sans-serif-medium"
+            android:fontFamily="Inter UI"
```

Comment:
Same here. CC @ekager @sblatz

Reply:
This one I saw used in `styles.xml` under `<style name="SearchShortcutsLabelStyle">`